### PR TITLE
Covering collaboration docs gaps 

### PR DESCRIPTION
**Pull request by arlemi**
This covers #2026

### CODE REVIEW

**Comment by arlemi** [comment]
@sankalp0o @claireriley @SueSmith There's a lot of commits but not that much changes, this PR is only about covering the gaps. I've left a few things out (for example: navigation from one page to the other at the bottom) since it doesn't make sense to do them with the general change of IA + reorg of Collaboration we're planning. 

Also pushing it now since I see I'm starting to have conflicts with other PRs that went in before mine... 👀

**Comment by SueSmith** [comment]
@claireriley wondering about maybe a more general redraft of the roles and permissions doc, any thoughts?

**Comment by claireriley** [comment]
> @claireriley wondering about maybe a more general redraft of the roles and permissions doc, any thoughts?

@SueSmith It definitely needs a major redraft, not sure if it's @arlemi 's domain or mine as it overlaps. @arlemi if you're up for it, perhaps a separate PR after this is merged? I'm happy to help, it's a pretty big project in itself to take on.

File: src/pages/docs/postman/collections/commenting-on-collections.md [hunk 1]
```diff
@@ -10,23 +9,60 @@ Postman provides a quick way to collaborate with your team members by allowing y
...
 
-**Note:** Postman app currently supports comments on requests only. Also, you can post comments only on saved requests.
+* Postman app currently supports comments on requests only.
+* Only those having access to a collection or a request will be able to post comments.
+* You can only post comments on saved requests.
 
-To post a comment, select the corresponding collection or a request and click **Comments**. The **Comments** dialog box appears. Write your comment and click *Add Comment*. Your comment is posted. The procedure to post a comment is similar both in the Postman app and the Dashboard. The following screen illustrates this:
+### Posting comments on a collection
+
+You can leave a comment on a collection through the [Postman Dashboard](https://app.getpostman.com). To post a comment on a collection, navigate to the `Collections` tab at the top of the page, select the collection you want to leave a comment on and then click the **Comments** button on the right of the collection title (circled in red in the picture below).
```
Comment (claireriley) [inline src/pages/docs/postman/collections/commenting-on-collections.md]: I'd change part of this to:
> select the collection you want to leave a comment on, then click **Comments**.

I'm not sure if the circle is necessary as it's red text (albeit a bit small). Unsure if we want to add this or an alternative highlighting option to the style guide @SueSmith?

File: src/pages/docs/postman/collections/commenting-on-collections.md [hunk 2]
```diff
@@ -10,23 +9,60 @@ Postman provides a quick way to collaborate with your team members by allowing y
...
+
+[![comment collection](https://user-images.githubusercontent.com/5029719/69899937-0d100600-1365-11ea-83c1-6d246b60cc54.png)](https://user-images.githubusercontent.com/5029719/69899937-0d100600-1365-11ea-83c1-6d246b60cc54.png)
+
+### Posting comments on a request
+
+You can post comments on requests from the [Postman Dashboard](https://app.getpostman.com) or the Postman app. When you post a comment on a request, it gets reflected in both the Dashboard and the app.
```
Comment (claireriley) [inline src/pages/docs/postman/collections/commenting-on-collections.md]: Wording suggestion:
> You can comment on requests by way of the [web dashboard](https://app.getpostman.com) or the Postman app. When you comment on a request, it is reflected in both the Dashboard and the app.

File: src/pages/docs/postman/collections/commenting-on-collections.md [hunk 3]
```diff
@@ -10,23 +9,60 @@ Postman provides a quick way to collaborate with your team members by allowing y
...
+
+To post a comment, select the corresponding request and click **Comments**. The **Comments** dialog box appears. Write your comment and click *Add Comment*. Your comment is posted. The procedure to post a comment is similar both in the Postman app and the Dashboard. The following screen illustrates this:
```
Comment (claireriley) [inline src/pages/docs/postman/collections/commenting-on-collections.md]: Bold "Add Comment", also possibility to condense to something like this:

> To post a comment, select the corresponding request and click **Comments**. Write your comment, then click **Add Comment** to publish.

File: src/pages/docs/postman/collections/commenting-on-collections.md [hunk 4]
```diff
@@ -10,23 +9,60 @@ Postman provides a quick way to collaborate with your team members by allowing y
...
 
 **Note:** If someone tags you in a comment on a private collection that you don’t have access for, Postman sends a notification to you. However, you can’t read the comment until the collection or the request is shared with you.  
 
+## Reading comments
+
+You can access the comments left on requests and collections by your teammates with the Postman app or the Postman dashboard.
+
+### Reading comments on the Postman app
+
+First, navigate to the request for which you want to read the comments by navigating to it using the left sidebar. Once you have opened on the correct request click on the **Comments** button (circled in red in the picture below) at the top-right corner of the request to see the existing comments.
```
Comment (claireriley) [inline src/pages/docs/postman/collections/commenting-on-collections.md]: I'd reduce to something like:

> To review comments, navigate to the request in question and click **Comments** to view all comments associated with the request.

File: src/pages/docs/postman/collections/commenting-on-collections.md [hunk 5]
```diff
@@ -10,23 +9,60 @@ Postman provides a quick way to collaborate with your team members by allowing y
...
+
+**Note:** The comment button displays the number of existing comments between brackets.
+
+[![read comments app](https://user-images.githubusercontent.com/5029719/70252300-1dffb380-1779-11ea-8317-3b9212e589f3.png)](https://user-images.githubusercontent.com/5029719/70252300-1dffb380-1779-11ea-8317-3b9212e589f3.png)
+
+### Reading comments on the Postman dashboard
+
+You can read the comments that have been left on a collection through the [Postman Dashboard](https://app.getpostman.com).
+
+To read the comments on a collection, navigate to the `Collections` tab at the top of the page, select the collection you want to see the comments of and then click the **Comments** button on the right of the collection title as shown in the animated picture below.
```
Comment (claireriley) [inline src/pages/docs/postman/collections/commenting-on-collections.md]: Option:

> To read the comments on a collection, navigate to the **Collections** tab at the top of the page, select the collection in mind, then click **Comments**.

File: src/pages/docs/postman/collections/commenting-on-collections.md [hunk 6]
```diff
@@ -10,23 +9,60 @@ Postman provides a quick way to collaborate with your team members by allowing y
...
+
+[![read comment collection dashboard](https://user-images.githubusercontent.com/5029719/71084214-b3913f00-218c-11ea-8796-cab07431775f.gif)](https://user-images.githubusercontent.com/5029719/71084214-b3913f00-218c-11ea-8796-cab07431775f.gif)
+
+To read the comments on a specific request, navigate to the `Collections` tab at the top of the page, select the collection to which the request belongs, click on that request and then click on the **Comments** button on the right of the request as shown in the animated picture below.
```
Comment (claireriley) [inline src/pages/docs/postman/collections/commenting-on-collections.md]: "click on that request and then click on the **Comments** button on the right of the request as shown in the animated picture below."
->
"select the request, then click **Comments**."

File: src/pages/docs/postman/design-and-develop-apis/managing-apis.md [hunk 7]
```diff
@@ -86,7 +86,37 @@ You will be able to perform all these user actions even from the dashboard, as i
 
 ## Commenting on APIs
 
-You can comment on your APIs the same way as you do on your collections. For more information on this, refer to [Commenting on Collections](/docs/postman/collections/commenting-on-collections/)
+You can comment on your APIs as you do on your collections. This can be done either on the [Postman Dashboard](https://app.getpostman.com) or with the Postman app.
```
Comment (claireriley) [inline src/pages/docs/postman/design-and-develop-apis/managing-apis.md]: Possible rewording: 
"This can be done by way of the [web dashboard](https://app.getpostman.com) or by way of the Postman app."

File: src/pages/docs/postman/design-and-develop-apis/managing-apis.md [hunk 8]
```diff
@@ -86,7 +86,37 @@ You will be able to perform all these user actions even from the dashboard, as i
...
+
+**Notes:**
+
+- Comments will be visible in both the dashboard and the app.
+- Only those having access to a collection or a request will be able to post comments.
```
Comment (claireriley) [inline src/pages/docs/postman/design-and-develop-apis/managing-apis.md]: Maybe:
"- Only those with access to a collection or a request may post comments."

also, should these be * instead of - ?

File: src/pages/docs/postman/design-and-develop-apis/managing-apis.md [hunk 9]
```diff
@@ -86,7 +86,37 @@ You will be able to perform all these user actions even from the dashboard, as i
...
+- You can use Markdown in your comments, for more information on formatting using Markdown refer to this [Markdown Cheatsheet on Github](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).
+
+### Commenting on the Postman Dashboard
+
+To leave a comment on an API using the Postman Dashboard, navigate to the API tab at the top of the page, select the API on which you want to make a comment and then click on the **Comments** button at the top-right of the page as illustred below. Once you have made your comment, click the *Add comment* button to publish it.
```
Comment (claireriley) [inline src/pages/docs/postman/design-and-develop-apis/managing-apis.md]: I'd remove all mentions of "button" and make sure all button labels are bolded (a few are italicized) in general if possible

File: src/pages/docs/postman/design-and-develop-apis/managing-apis.md [hunk 10]
```diff
@@ -86,7 +86,37 @@ You will be able to perform all these user actions even from the dashboard, as i
...
+
+[![api comment dashboard](https://user-images.githubusercontent.com/5029719/69900515-48fa9980-136c-11ea-9e55-7785626dfcad.gif)](https://user-images.githubusercontent.com/5029719/69900515-48fa9980-136c-11ea-9e55-7785626dfcad.gif)
+
+### Commenting on the Postman app
+
+To leave a comment on an API using the Postman app, select the API on which you want to make a comment and then click on the **Comments** button at the top-right of the page as illustred below. Once you have made your comment, click the *Add comment* button to publish it.
```
Comment (claireriley) [inline src/pages/docs/postman/design-and-develop-apis/managing-apis.md]: ~~button~~

File: src/pages/docs/postman/workspaces/creating-workspaces.md [hunk 11]
```diff
@@ -42,9 +42,9 @@ Postman account users can create an unlimited number of personal workspaces. Use
 
 ## How to create a new workspace?
 
-Click the workspace dropdown menu that displays in the header bar. For example, in the image below, you'd click the workspace called "My Workspace". Click the **Create New** link. Determine whether you'd like to create a personal or team workspace.
+Click the workspace dropdown menu that displays in the header bar. For example, in the image below, you'd click the workspace called "My Workspace". Click the **Create New** link (circled in red in the image below). Determine whether you'd like to create a personal or team workspace.
```
Comment (claireriley) [inline src/pages/docs/postman/workspaces/creating-workspaces.md]: Could possibly be condensed to:

> To create a new workspace, select the workspace dropdown menu > **Create New**.

File: src/pages/docs/postman/collections/commenting-on-collections.md (hunk 1 above)
Comment (SueSmith) [reply src/pages/docs/postman/collections/commenting-on-collections.md]: Yep I tend to avoid circling etc to keep product pics clean, at some stage we could maybe get more fixed guidelines for images with input from design..

File: src/pages/docs/postman/collections/commenting-on-collections.md [hunk 12]
```diff
@@ -47,8 +83,12 @@ Sometimes you may want to call the attention of a team member when posting a com
 
 [![post comments](https://assets.postman.com/postman-docs/comment-mention.png)](https://assets.postman.com/postman-docs/comment-mention.png)
 
+The user tagged in your comment will get a notification by email and/or in-app as long as they have not disabled these types of notifications. The notifications in-app are shown at the top-right on the bell icon as pictured below. Clicking on the name of the collection will take them straight to the comment.
```
Comment (SueSmith) [inline src/pages/docs/postman/collections/commenting-on-collections.md]: I'd get rid of "as pictured below" and reword final sentence to something like "To navigate to the comment, the recipient can click the collection name."

File: src/pages/docs/postman/collections/commenting-on-collections.md [hunk 13]
```diff
@@ -47,8 +83,12 @@ Sometimes you may want to call the attention of a team member when posting a com
...
+
+[![receive comment](https://user-images.githubusercontent.com/5029719/71098559-cb2af080-21a9-11ea-95ae-68b5dbb2eea9.png)](https://user-images.githubusercontent.com/5029719/71098559-cb2af080-21a9-11ea-95ae-68b5dbb2eea9.png)
+
 ## Markdown support
 
-Postman comments support Markdown. You can format the text the way you want in the comments dialog box. The following screen illustrates formatting bold faced and italicized text in the comments dialog box:
+Postman comments support Markdown. You can format the text the way you want in the comments dialog box. For more information on formatting using Markdown refer to [Markdown in API Documentation](https://documenter.getpostman.com/view/33232/markdown-in-api-documentation/JsGc?version=latest). The following screen illustrates formatting bold faced and italicized text in the comments dialog box:
```
Comment (SueSmith) [inline src/pages/docs/postman/collections/commenting-on-collections.md]: I'd lose the sentences "You can format..." and "The following screen"

File: src/pages/docs/postman/collections/sharing-collections.md [hunk 14]
```diff
@@ -58,7 +60,7 @@ The **SHARE COLLECTION** modal appears. It offers three ways to share a collecti
 * Embed
 * Get Link
 
-[![share sidebar](https://assets.postman.com/postman-docs/Collection_Share-Sidebar.png)](https://assets.postman.com/postman-docs/Collection_Share-Sidebar.png)
+[![share sidebar](https://user-images.githubusercontent.com/5029719/71100430-eb0fe380-21ac-11ea-9ad6-1c7922f4f722.png)](https://user-images.githubusercontent.com/5029719/71100430-eb0fe380-21ac-11ea-9ad6-1c7922f4f722.png)
```
Comment (SueSmith) [inline src/pages/docs/postman/collections/sharing-collections.md]: as with circles etc, I'd probably lose the arrow

File: src/pages/docs/postman/design-and-develop-apis/managing-apis.md (hunk 7 above)
Comment (SueSmith) [reply src/pages/docs/postman/design-and-develop-apis/managing-apis.md]: I'd lose the passive and just rephrase the whole paragraph:

"You can comment on APIs in the Postman app or Dashboard."

File: src/pages/docs/postman/design-and-develop-apis/managing-apis.md [hunk 15]
```diff
@@ -86,7 +86,37 @@ You will be able to perform all these user actions even from the dashboard, as i
...
-You can comment on your APIs the same way as you do on your collections. For more information on this, refer to [Commenting on Collections](/docs/postman/collections/commenting-on-collections/)
+You can comment on your APIs as you do on your collections. This can be done either on the [Postman Dashboard](https://app.getpostman.com) or with the Postman app.
+
+**Notes:**
+
+- Comments will be visible in both the dashboard and the app.
+- Only those having access to a collection or a request will be able to post comments.
+- You can use Markdown in your comments, for more information on formatting using Markdown refer to this [Markdown Cheatsheet on Github](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).
+
+### Commenting on the Postman Dashboard
```
Comment (SueSmith) [inline src/pages/docs/postman/design-and-develop-apis/managing-apis.md]: I'd change to "in" rather than "on"

File: src/pages/docs/postman/design-and-develop-apis/managing-apis.md [hunk 16]
```diff
@@ -86,7 +86,37 @@ You will be able to perform all these user actions even from the dashboard, as i
...
+
+To leave a comment on an API using the Postman Dashboard, navigate to the API tab at the top of the page, select the API on which you want to make a comment and then click on the **Comments** button at the top-right of the page as illustred below. Once you have made your comment, click the *Add comment* button to publish it.
+
+[![api comment dashboard](https://user-images.githubusercontent.com/5029719/69900515-48fa9980-136c-11ea-9e55-7785626dfcad.gif)](https://user-images.githubusercontent.com/5029719/69900515-48fa9980-136c-11ea-9e55-7785626dfcad.gif)
+
+### Commenting on the Postman app
```
Comment (SueSmith) [inline src/pages/docs/postman/design-and-develop-apis/managing-apis.md]: again with "in" instead of "on"

File: src/pages/docs/postman/design-and-develop-apis/managing-apis.md [hunk 17]
```diff
@@ -86,7 +86,37 @@ You will be able to perform all these user actions even from the dashboard, as i
...
+
+[![api comment app](https://user-images.githubusercontent.com/5029719/69900530-6a5b8580-136c-11ea-9834-e5f8eee85ce7.gif)](https://user-images.githubusercontent.com/5029719/69900530-6a5b8580-136c-11ea-9834-e5f8eee85ce7.gif)
+
+### Tagging members in a comment
+
+Sometimes you may want to call the attention of a team member when posting a comment. In such a scenario, you can tag a person by typing the @ symbol followed by the member’s name and then choosing the corresponding name from the auto-suggested list. When you tag a member in your comment, Postman notifies the member by sending a notification.
+
+### Editing and deleting comments
+
+To edit or delete one of your comment, navigate to the comment section and hover with your mouse on the comment you wish to manage. Click on the pen icon to edit the comment or click on the bin icon to delete the comment.
```
Comment (SueSmith) [inline src/pages/docs/postman/design-and-develop-apis/managing-apis.md]: * wee typo "comments" instead of "comment" in first clause
* I'd also remove "with your mouse", just "hover over the comment" should be fine
* remove "on" from "click on"

File: src/pages/docs/postman/design-and-develop-apis/managing-apis.md [hunk 18]
```diff
@@ -86,7 +86,37 @@ You will be able to perform all these user actions even from the dashboard, as i
...
+
+[![api comment manage](https://user-images.githubusercontent.com/5029719/69900613-9e837600-136d-11ea-886a-6c1dd0de085c.png)](https://user-images.githubusercontent.com/5029719/69900613-9e837600-136d-11ea-886a-6c1dd0de085c.png)
+
+**Note:** For moderation purposes, Admins can delete anyone’s comment. However, admins will not be able to update/modify a comment.
```
Comment (SueSmith) [inline src/pages/docs/postman/design-and-develop-apis/managing-apis.md]: again with blockquote for note, would maybe reword slightly "With Admin permissions you can delete comments made by any contributor, but you cannot update or modify comments."

File: src/pages/docs/postman/workspaces/creating-workspaces.md [hunk 19]
```diff
@@ -56,8 +56,8 @@ Postman Enterprise users can check the box to limit the visibility of this works
 
 Click the **Create Workspace** button to finish creating your workspace.
 
-[![create new team workspace](https://assets.postman.com/postman-docs/Workspace_team_create_new.png)](https://assets.postman.com/postman-docs/Workspace_team_create_new.png)
+[![create new team workspace](https://user-images.githubusercontent.com/5029719/71102959-36c48c00-21b1-11ea-9a18-b5038af699ca.png)](https://user-images.githubusercontent.com/5029719/71102959-36c48c00-21b1-11ea-9a18-b5038af699ca.png)
 
-You can also create a new workspace in the [Workspaces dashboard](https://app.getpostman.com/dashboard). Click the **Create a new workspace** button and follow the steps on how to create a personal or team workspace above.
+You can also create a new workspace in the [Workspaces dashboard](https://app.getpostman.com/dashboard). Click the **Create a new workspace** button (circled in red in the image below) and follow the steps on how to create a personal or team workspace above.
```
Comment (SueSmith) [inline src/pages/docs/postman/workspaces/creating-workspaces.md]: lose "(circled in red in the image below)"

File: src/pages/docs/postman/workspaces/sharing-collections-in-workspaces-for-version-5.md [hunk 20]
```diff
@@ -20,6 +20,7 @@ contextual_links:
 warning: false
 ---
 
+**Note:** If you're looking to share a collection on version 6 and higher, refer to the [Sharing collections documentation](/docs/postman/collections/sharing-collections/).
```
Comment (SueSmith) [inline src/pages/docs/postman/workspaces/sharing-collections-in-workspaces-for-version-5.md]: same again with blockquote for note

File: src/pages/docs/postman/collections/commenting-on-collections.md [hunk 21]
```diff
@@ -1,6 +1,5 @@
 ---
 title: "Commenting on collections"
-order: 38
```
[the remark this answers is not in the document]
Comment (arlemi) [reply src/pages/docs/postman/collections/commenting-on-collections.md]: That's my bad, removed it when trying to think of how to reorganise the whole thing and forgot to put it back when I gave up. 🙃

File: src/pages/docs/postman/collections/commenting-on-collections.md [hunk 22]
```diff
@@ -10,23 +9,60 @@ Postman provides a quick way to collaborate with your team members by allowing y
...
 * [Tagging members in comments](#tagging-members-in-comments)
 * [Markdown support](#markdown-support)
 
 ## Posting comments
 
-You can post comments on your collections and requests from the Postman Dashboard. Only those having access to a collection or a request will be able to post comments. When you post a comment on a collection or a request, it gets reflected in both the Postman app and the Dashboard.
+**Notes:**
 
-**Note:** Postman app currently supports comments on requests only. Also, you can post comments only on saved requests.
+* Postman app currently supports comments on requests only.
```
[the remark this answers is not in the document]
Comment (arlemi) [reply src/pages/docs/postman/collections/commenting-on-collections.md]: Changed the second one but keeping the first one since we want to emphasise on the difference between using the app and the dashboard which for commenting still provide different features as of now. Saying "Postman currently supports on requests only" may let the user believe this include on the web as well.

File: src/pages/docs/postman/design-and-develop-apis/managing-apis.md (hunk 8 above)
Comment (arlemi) [reply src/pages/docs/postman/design-and-develop-apis/managing-apis.md]: I've had the linter picking up on `-` and/or on `*` for lists, still trying to figure out what's the case for each 🤔

**Comment by arlemi** [comment]
@claireriley @SueSmith Thanks for all the corrections/suggestions, I think I've removed most/all:
- references to screenshots
- red circles/arrows on screenshots
- wrong formatting (*italic* instead of **bold**)
- "button" occurrences

as well as included most of your suggestions because it just sounds better anyway. 😄

File: src/pages/docs/postman/design-and-develop-apis/managing-apis.md [hunk 23]
```diff
@@ -16,31 +16,31 @@ APIs reside in your workspace, just like your collections. This section walks yo
...
 ## Creating an API
 
-You can create an API in the workspace from the sidebar in the Postman app and from the workspaces dashboard. To create an API in the Postman app when no APIs exist in the current workspace, click **Create an API** button in the sidebar, as illustrated in a red circle in the following screen:
+You can create an API in the workspace from the sidebar in the Postman app and from the workspaces dashboard. To create an API in the Postman app when no APIs exist in the current workspace, click **Create an API** in the sidebar.
 
-[![api create4](https://assets.postman.com/postman-docs/API-Create4.png)](https://assets.postman.com/postman-docs/API-Create4.png)
+[![api create4](https://user-images.githubusercontent.com/5029719/71972999-bcae8280-3205-11ea-81f3-1482988b7426.png)](https://user-images.githubusercontent.com/5029719/71972999-bcae8280-3205-11ea-81f3-1482988b7426.png)
 
-If you already have created APIs in your workspace, click '+' in the sidebar, as indicated by the red arrow in the above screen. The following tab opens:
+If you already have created APIs in your workspace, click '+' in the sidebar, as indicated by the red arrow in the above screen.
```
Comment (SueSmith) [inline src/pages/docs/postman/design-and-develop-apis/managing-apis.md]: one wee use of "...the above screen" we missed 🙈

Comment (arlemi) [reply src/pages/docs/postman/design-and-develop-apis/managing-apis.md]: Fixed :D
